wurst/systems/units/buildings/Armory.wurst

```diff
@@ -7,7 +7,7 @@ import UnitObjEditing
 import ObjectIds
 
 
-constant MODEL       = "Models\\SlaughterHouse.mdx"
+constant MODEL       = "Models\\Armory.mdx"
```

Comment:
Add a new class to `LocalAssets` called `LocalBuildings` and add this there, emulating https://github.com/wurstscript/WurstStdlib2/blob/master/wurst/_wurst/assets/Buildings.wurst#L4